javascript-practice/src/styles/components/product-item.css

```diff
@@ -0,0 +1,40 @@
+.product-item {
+  --color-light: #d9d9d9;
+  --color-black: #272727;
+}
+
+.product-thumbnail {
+  padding: 45px 17px;
+}
+
+.product-name,
+.product-price {
```

Comment:
You can use a class here instead use multiple classes that have the same style. Example .product-info {......}